Makes second set of changes to IPI docs (#113832)
openshift/openshift-docs: modules/nw-network-config.adoc

[id="nw-network-config_{context}"]
= Network configuration phases

[role="_abstract"]

> You can use this reference to plan network customizations for {product-title} before you install the cluster. Customize settings in the `install-config.yaml` file and in the Cluster Network Operator manifest across two configuration phases.

Reviewer: This is another instance where the short description is self referential, can this be rewritten? IMO, the existing intro isn't too bad, maybe you can keep that as the first sentence and add the second sentence of your new rewrite if you want to add that additional context?
```suggestion
There are two phases prior to {product-title} installation where you can customize the network configuration. Customize settings in the `install-config.yaml` file and in the Cluster Network Operator manifest across two configuration phases.
```
